package lv.diana.module;

public class TestExceptionHandling {

	
	public static void main(String[] args) {
		try {
			int number1 = 0;
			int number2 = 10;
			int result = number2 / number1;
			
			System.out.println(result);
			
		} catch (Exception e) {
			// TODO: handle exception
			System.out.println("Check code in try block" +e);
		}
		
		
		

	}

}
